src/components/ChatHeader/ChatHeader.jsx

```diff
@@ -1,10 +1,36 @@
+import { useState } from 'react';
+import CountryInfo from '../CountryInfo/CountryInfo';
 import { ChatHeaderStyled } from './ChatHeaderStyled';
 
-const ChatHeader = ({ countryName, participantsAmount }) => (
-  <ChatHeaderStyled>
-    <h5>{countryName || 'Country Name'}</h5>
-    <p>{`${participantsAmount || '0'} participants`}</p>
-  </ChatHeaderStyled>
-);
+const ChatHeader = ({
+  countryName,
+  participantsAmount,
+  countryChatId,
+  setSubscriptionCountryRooms,
+  isSubscribed,
+}) => {
+  const [openModal, setOpenModal] = useState(false);
+  const handleOpen = () => setOpenModal(true);
+  const handleClose = () => {
+    setOpenModal(false);
+  };
+  return (
+    <>
+      <ChatHeaderStyled onClick={handleOpen}>
+        <h5>{countryName || 'Country Name'}</h5>
```

Comment:
Use default params to avoid such conditions https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Default_parameters.

Reply:
Done. I hope I understood you correctly
`const ChatHeader = ({
  countryName = 'Country Name',
  participantsAmount = 0,`

` <h5>{countryName}</h5>
        <p>{`${participantsAmount} participants`}</p>`